dco-sample.ttl

```diff
@@ -0,0 +1,184 @@
+@prefix rdfs:    <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix vivo:    <http://vivoweb.org/ontology/core#> .
+@prefix obo:     <http://purl.obolibrary.org/obo/> .
+@prefix dco:     <http://info.deepcarbon.net/schema#> .
+@prefix xsd:     <http://www.w3.org/2001/XMLSchema#> .
+@prefix core:    <http://vivoweb.org/ontology/core#> .
+@prefix skos:    <http://www.w3.org/2004/02/skos/core#> .
+@prefix ero:     <http://purl.obolibrary.org/obo/> .
+@prefix bibo:    <http://purl.org/ontology/bibo/> .
+@prefix foaf:    <http://xmlns.com/foaf/0.1/> .
+
+<http://info.deepcarbon.net/sample/schema>
+      a       owl:Ontology ;
+      rdfs:label "DCO Sample"@en-US .
+
+dcosample:inSampleRepository
+      a       owl:ObjectProperty ;
+      rdfs:domain dcosample:Sample ;
+      rdfs:label "in sample repository"@en-US ;
+      rdfs:range dcosample:SampleRepository ;
+      rdfs:comment  """
+      """^^rdf:XMLLiteral .
+
+dcosample:IGSNRegistration
+      a       owl:Class ;
+      rdfs:label "IGSN Registration"@en-US ;
+      rdfs:comment  """
+          information about the registration of an IGSN
+      """^^rdf:XMLLiteral .
+
+dcosample:igsnRegistrant
+      a       owl:ObjectProperty ;
+      rdfs:domain dcosample:IGSNRegistration ;
+      rdfs:label "registered by"@en-US ;
+      rdfs:range foaf:Person ;
+      rdfs:comment  """
+          registration of the sample to get igsn was done by a person
+      """^^rdf:XMLLiteral .
+
+dcosample:includesSampleType
+      a       owl:ObjectProperty ;
+      rdfs:domain dcosample:SampleGroup ;
+      rdfs:label "includes sample type"@en-US ;
+      rdfs:range dcosample:SampleType ;
+      rdfs:comment  """
+          A collection of samples includes samples of various types
+      """^^rdf:XMLLiteral .
+
+dcosample:hasSampleType
+      a       owl:ObjectProperty ;
+      rdfs:domain dcosample:Sample ;
+      rdfs:label "has sample type"@en-US ;
+      rdfs:range dcosample:SampleType ;
+      rdfs:subPropertyOf core:hasSubjectArea ;
+      rdfs:comment  """
+          A sample classification or annotation
+      """^^rdf:XMLLiteral .
+
+dcosample:hasIGSNRegistration
+      a       owl:ObjectProperty ;
+      rdfs:domain samfl:Specimen ;
+      rdfs:label "has IGSN registration"@en-US ;
+      rdfs:range dcosample:IGSNRegistration ;
+      rdfs:comment  """
+          information about the registration of the sample such as registering, removing the registration, discarding the registration
+      """^^rdf:XMLLiteral .
+
+dcosample:includesSampleGroup
+      a       owl:ObjectProperty ;
+      rdfs:domain dcosample:SampleRepository ;
+      rdfs:label "includes sample group"@en-US ;
+      rdfs:range dcosample:SampleGroup ;
+      rdfs:comment  """
+          a sample repository can include collections of samples as well as individual samples
+      """^^rdf:XMLLiteral .
+
+dcosample:SampleGroup
+      a       owl:Class ;
+      rdfs:label "Sample Group"@en-US ;
+      rdfs:subClassOf dco:Object ;
```

Comment:
Should this also be a subclass of [``samfl:SamplingCollection``?](http://def.seegrid.csiro.au/ontology/om/sam-lite#SamplingCollection)